source/scaling.rst

```diff
@@ -6,21 +6,35 @@ platform is the ease of an application scaling. Scaling a Deployment up
 or down ensures new Pods are created and set to available Kubernetes
 nodes.
 
-Size of the cluster is controlled by a :ref:`size key<replsets-size>` in the :ref:`operator.custom-resource-options` configuration.. That’s why scaling the cluster needs
-nothing more but changing this option and applying the updated
-configuration file. This may be done in a specifically saved config, or
-on the fly, using the following command, which saves the current
-configuration, updates it and applies the changed version:
+The size of the cluster is controlled by the ``size`` key in the
+:ref:`operator.custom-resource-options` configuration.
 
-.. code:: bash
+.. note:: Using ``kubectl scale StatefulSet_name`` command to rescale
+   Percona Server for MongoDB is not recommended, as it makes ``size``
+   configuration options out of sync, and the next config change may result
+   in reverting the previous number of nodes.
```

Comment:
This is not correct.
1. Operator will not allow the user to scale statefulset and will revert to the original size in the next reconcile loop (5 seconds).
2. The command for scaling StatefulSet is: kubectl scale statefulset StatefulSet_name